ModifiedBayes.py

```diff
@@ -0,0 +1,168 @@
+
+	'''Assigns the keys "higher" and "lower" to the two possible sensitive attribute values based on which of the two has a higher count.
+	   S+ ("higher") is the privileged group. We do this based on counts instead of as a manual parameter because there isn't an 'ideal' 
+	   sensitive attribute category like there is with classifications.'''
+	def assignSensitivity(self, dataSet, dataFrame, sensitivityDict):
+		sensitiveAttrCatList = self.getAttributeCategories(dataFrame, dataSet.protectedAttribute)
+		Sx = dataFrame.loc[dataFrame[dataSet.protectedAttribute] == sensitiveAttrCatList[0], dataSet.protectedAttribute].count()
+		Sy = dataFrame.loc[dataFrame[dataSet.protectedAttribute] == sensitiveAttrCatList[1], dataSet.protectedAttribute].count()
+		if (Sx > Sy):
+			sensitivityDict["higher"] = sensitiveAttrCatList[0]
+			sensitivityDict["lower"] = sensitiveAttrCatList[1]
+		else:
+			sensitivityDict["higher"] = sensitiveAttrCatList[1]
+			sensitivityDict["lower"] = sensitiveAttrCatList[0]
+
+	'''Counts up the number of elements in a particular column that match the classification value located in the classDict passed in 
+	   with the key "higher" (AKA - C+).'''
+	def calculateNumPos(self, dataFrame, column, classDict):
+		return dataFrame.loc[dataFrame[column] == classDict["higher"], column].count()
+
+	'''A function that can be called in the while loop to keep track/ watch how the counts are changing with each iteration'''
+	def printCounts(self, dataSet, CHigherSLowerCount, CLowerSLowerCount, CHigherSHigherCount, CLowerSHigherCount, higherOrLowerSensitiveAttributeDict, higherOrLowerClassificationDict):
+		dataFrame = dataSet.dataFrame
+		print("c+s- count:", CHigherSLowerCount)
+		print("c-s- count:", CLowerSLowerCount)
+		print("c+s+ count:", CHigherSHigherCount)
+		print("c-s+ count:", CLowerSHigherCount)
+		print("bayes classification column c+s- count: ", self.countIntersection(dataFrame, dataSet.protectedAttribute, higherOrLowerSensitiveAttributeDict["lower"], "Bayes Classification", higherOrLowerClassificationDict["higher"]))
+		print("bayes classification column c-s- count: ", self.countIntersection(dataFrame, dataSet.protectedAttribute, higherOrLowerSensitiveAttributeDict["lower"], "Bayes Classification" , higherOrLowerClassificationDict["lower"]))
+		print("bayes classification column c+s+ count: ", self.countIntersection(dataFrame, dataSet.protectedAttribute, higherOrLowerSensitiveAttributeDict["higher"], "Bayes Classification", higherOrLowerClassificationDict["higher"]))
+		print("bayes classification column c-s+ count: ", self.countIntersection(dataFrame, dataSet.protectedAttribute, higherOrLowerSensitiveAttributeDict["higher"], "Bayes Classification", higherOrLowerClassificationDict["lower"]))
+
+	'''Space saving function for modify() that prints out probabilities'''
+	def printProbabilities(self, CHigherSLower, CLowerSLower, CHigherSHigher, CLowerSHigher):
+		print("c+s-:", CHigherSLower)
```

Comment:
maybe add in prob to print statement as you did for count above to make it more clear to people reading output in terminal